src/app/state/accountHistorySlice.ts

```diff
@@ -26,11 +26,13 @@ export interface AccountHistoryState {
 const initialState: AccountHistoryState = {
   trades: [],
   orderHistory: [],
+  // these tables are component specific and should not be in app state
   selectedTable: Tables.OPEN_ORDERS,
   tables: Object.values(Tables),
 };
 
 // ASYNC THUNKS
+// thunks should change state or call other functions that change state. This is just a normal async function that fetches data.
```

Comment:
Can you explain why account history fecthing does not change the state? As far as I understand those trades done by the user should be stored in global state (effectively updating state once the data is fetched).

Reply:
Your right. When I was adding these coomments I must have missed that there is an associated extraReducer for this thunk that does change state.